<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDossierDialog</class>
 <widget class="QWidget" name="CreateDossierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="patientGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>1131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: 75 12pt &quot;Century Schoolbook L&quot;;
  border: 1px solid gray;
  border-radius: 9px;
  margin-top: 0.5em;
  background: transparent;
}

QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; 
}

QLabel {
background: transparent;
}

QCheckBox {
background: white;
}


</string>
   </property>
   <property name="title">
    <string>Patient</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1081</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lastNameLabel_2">
       <property name="text">
        <string>Dossier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dossierLineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>Prénom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstNameLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="patientGroupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>871</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: 75 12pt &quot;Century Schoolbook L&quot;;
  border: 1px solid gray;
  border-radius: 9px;
  margin-top: 0.5em;
  background: transparent;
}

QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; 
}

QLabel {
background: transparent;
}

QCheckBox {
background: white;
}


</string>
   </property>
   <property name="title">
    <string>Type de traitement</string>
   </property>
   <widget class="QTableWidget" name="tableWidget_treatment_types">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>831</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>20</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>250</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>234</red>
       <green>234</green>
       <blue>234</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dose intégrée</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>234</red>
       <green>234</green>
       <blue>234</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Séances</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>234</red>
       <green>234</green>
       <blue>234</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Boost</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>230</red>
       <green>230</green>
       <blue>230</blue>
      </color>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createDossierPushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>470</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Créer</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="patientGroupBox_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>140</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: 75 12pt &quot;Century Schoolbook L&quot;;
  border: 1px solid gray;
  border-radius: 9px;
  margin-top: 0.5em;
  background: transparent;
}

QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; 
}

QLabel {
background: transparent;
}

QCheckBox {
background: white;
}


</string>
   </property>
   <property name="title">
    <string>Ajouter séance</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Dose [Gy]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_seance_dose">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addSeance">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addBoost">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Boost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_resetTable">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Réinitialiser</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dossierLineEdit</tabstop>
  <tabstop>lastNameLineEdit</tabstop>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>tableWidget_treatment_types</tabstop>
  <tabstop>lineEdit_seance_dose</tabstop>
  <tabstop>pushButton_addSeance</tabstop>
  <tabstop>pushButton_addBoost</tabstop>
  <tabstop>pushButton_resetTable</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>createDossierPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
